acceptance-tests/lib/cmd.ts

```diff
@@ -64,12 +60,10 @@ function executeWithInput(
     inputs = [];
   }
 
-  // Prevent the browser from opening when `open` is called
-  opts.env = { BROWSER: 'none' };
```

Comment:
I moved this into the `spawn()` call above because I think it makes more sense there. This will also prevent us from overriding any existing `opts.env` values that get passed into `executeWithInput()`